android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java

```diff
@@ -605,6 +632,120 @@ public void didSelectTab(Tab tab, @TabSelectionType int type, int lastId) {
         };
     }
 
+    private void hidePlaylistButton() {
+        if (BraveActivity.getBraveActivity() != null) {
```

Comment:
getBraveActivity needs to handle exception.

Reply:
Yes, `getBraveActivity` throws the exception now. Btw at current release 1.49.x - it doesn't throw, so the change is quite new.